src/components/Home/Home.tsx

```diff
@@ -0,0 +1,223 @@
+import Image from 'next/image';
+import Link from 'next/link';
+
+import { ROUTES } from '@/shared/constants/routes';
+
+import Badge from '../Badge/Badge';
+import SvgIconChevronRight from '../Icon/IconChevronRight';
+import SvgIconPairQuotation from '../Icon/IconPairQuotation';
+import AnalyzeCard from './AnalyzeCard';
+import FeedbackForm from './FeedbackForm';
+import Looper from './Looper';
+import StartButton from './StartButton';
+
+const ANALYZE_CARD = ['직무 선택', '경험 작성', '직무 역량 추진', '경험카드 생성'];
+
+const Home = () => {
+  return (
+    <>
+      <main>
+        <section className="flex flex-col justify-center items-center pt-[230px] pb-[63px] bg-white">
+          <h1 className="text-center text-[60px] font-bold leading-[80px] whitespace-pre-line mb-[26px]">
+            {'자기소개서가 막막한 순간,\n그 시작을 함께하는\n인사이트 아웃'}
+          </h1>
+          <p className="text-center whitespace-pre-line mb-[52px]">
+            {
+              '인사이트 아웃을 통해 질문으로 나의 프로젝트를 정리하며 경험을 파헤치고\n나만의 경험 역량 카드로 만들어 자기소개서를 작성해보세요.'
+            }
+          </p>
+          <StartButton route={ROUTES.EXPERIENCE}>경험 분해 시작하기</StartButton>
+        </section>
+        <section className="relative h-[1014px] bg-white">
+          <Image src={'/images/home/img-home-1.png'} fill alt="Home-1" className="object-contain w-full h-full" />
+        </section>
+        <section className="relative flex flex-col justify-center items-center pt-[170px] pb-[283px] bg-white z-[-1]">
+          <div className="flex justify-center items-center mb-[286px] gap-[102px]">
+            <div className="flex flex-col gap-[44px]">
+              <div className="flex flex-col w-[588px] gap-[30px]">
+                <Badge variant="primary50-outline" size="L">
+                  경험분해 4 STEPS
+                </Badge>
+                <h1 className="text-main text-[44px] font-bold whitespace-pre-line">
+                  {'질의응답으로\n 경험을 분해해요'}
+                </h1>
```

Comment:
```suggestion
                <h2 className="text-main text-[44px] font-bold whitespace-pre-line">
                  {'질의응답으로\n 경험을 분해해요'}
                </h2>
```

하나의 HTML에는 하나의 h1만 사용하는 것이 좋다고 합니다~

> [heading mdn](https://developer.mozilla.org/ko/docs/Web/HTML/Element/Heading_Elements#%EC%82%AC%EC%9A%A9_%EC%9D%BC%EB%9E%8C)
페이지 당 하나의 `<h1>`만 사용하세요. 여러 개를 써도 오류는 나지 않겠지만, 단일 `<h1>`이 모범 사례로 꼽힙니다. 논리적으로 생각했을 때도, `<h1>`은 가장 중요한 제목이므로 전체 페이지의 목적을 설명해야 할 것입니다. 두 개의 제목을 가진 책이나, 여러 개의 이름을 가진 영화는 볼 수 없죠! 또한 스크린 리더 사용자와 [SEO](https://developer.mozilla.org/ko/docs/Glossary/SEO)에도 더 적합합니다.

Reply:
ㄷㄷ 잡아내주셔서 감사합니다..